index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
+        <div class="box header">
+            <span>My Logo</span>
+            <span><img src="img/Enabled.png"></span>
+        </div>
+        <div class="box content">
+            <div class="title">
+            I’m Mariane 
+            Glad to see 
+            you!
+            </div>
+            <br>
+            <div class="paragraph">
+                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! 
+                   If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </div>
+            <br>
+            <div class="connect">
+                LET’S CONNECT
+            </div>
+            <br>
+            <div class="network-icons">
+                <a href="https://twitter.com" target="_blank"><img src="img/TwitterIcon.png" alt=""></a>
+                <a href="https://linkedin.com" target="_blank"><img src="img/LinkedinIcon.jpg" alt=""></a>
+                <a href="https://medium.com" target="_blank"><img src="img/MediumIcon.jpg" alt=""></a>
+                <a href="https://github.com" target="_blank"><img src="img/GithubIcon.jpg" alt=""></a>
+                <a href="https://youtube.com" target="_blank"><img src="img/HandIcon.jpg" alt=""></a>
+            </div>
```

Comment:
- [ ] Great job here 💯 However, It would be better to use `<ul>` and `<li>` tags, The list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).

> For example 💡
```
<ul class="network-icons">
    <li><a href="https://twitter.com" target="_blank"><img src="img/TwitterIcon.png" alt=""></a></li>
    <li> <a href="https://linkedin.com" target="_blank"><img src="img/LinkedinIcon.jpg" alt=""></a></li>
     ....
</ul>
```